### PR TITLE
Add clarity to confirmation dialog text when deleting course users and invitations

### PR DESCRIPTION
**Pull request by aylz**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (zhuhanming) [inline yarn.lock:1]: Please remove this file - you should not be doing `yarn install` in the root, only in `client`.

Comment (aylz) [reply yarn.lock]: done!

**Review by zhuhanming — CHANGES_REQUESTED** [review]
Small nit: Can you lowercase the `role` of the user? I find it slightly strange that it's capitalised halfway through the sentence. Same for the student/staff deletion - I feel that it looks better with `Delete student:` than `Delete Student:`.

Screenshot 2021-09-30 at 10 03 26 PM

**Comment by zhuhanming** [comment]
@aylz you committed your `yarn.lock` and your schema changes as well - can remove? Try to `git add` specific files if possible next time.

**Comment by aylz** [comment]
> @aylz you committed your `yarn.lock` and your schema changes as well - can remove? Try to `git add` specific files if possible next time.

ah sorry - i'll make sure that doesn't happen the next time!

**Review by zhuhanming — APPROVED** [review]
LGTM!

**Pull request merged**